snippets/accessibility/AccessibleLinks.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Snippet Title: Accessible Link Colors
+ * Scope:         global

```

Comment:
Your snippet uses the `.pagelayout-incourse, .pagelayout-course` selectors. But here you are saying `Scope: global` and in the description you are saying `across the site`.

I am wondering what intention really is. If this snippet should have a global scope, then the CSS selectors are wrong. If this snippet should have a course scope, then the comments here should be fixed.
I would be fine with both.